insights/scripts/foodtruck/Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pylint = "*"
+
+[packages]
+boto3 = "*"
+
+[requires]
+python_version = "3.8"
```

Comment:
There was some discussion here as to whether the pipfile standard was being maintained: https://github.com/CMSgov/beneficiary-fhir-data/pull/187

Reply:
Interesting. I've been using pipenv because it is similar to NPM. Nevertheless, I'll go to requirements.txt.

Reply:
Added a requirements.txt file.